include/boost/geometry/strategies/geographic/distance_cross_track.hpp

```diff
@@ -307,17 +337,18 @@ private :
                        CT const& lon3, CT const& lat3, //query point p3
                        Spheroid const& spheroid,
                        CT const& s14_start, CT const& a12,
-                       result_distance_point_segment<CT>& result)
+                       result_type<CT>& result)
     {
-        typedef typename FormulaPolicy::template inverse<CT, true, true, false, true, true>
-                inverse_distance_azimuth_quantities_type;
-        typedef typename FormulaPolicy::template inverse<CT, false, true, false, false, false>
-                inverse_dist_azimuth_type;
-        typedef typename FormulaPolicy::template direct<CT, true, false, false, false>
-                direct_distance_type;
+        using inverse_distance_azimuth_quantities_type = typename 
+            FormulaPolicy::template inverse<CT, true, true, false, true, true>;
```

Comment:
So here the same thing is formatted differently.

Reply:
changed to 
```
using inverse_distance_azimuth_quantities_type = 
            typename FormulaPolicy::template inverse<CT, true, true, false, true, true>;
    
using inverse_dist_azimuth_type = 
            typename FormulaPolicy::template inverse<CT, false, true, false, false, false>;

using direct_distance_type = 
            typename FormulaPolicy::template direct<CT, true, false, false, false>;
```